### DISCUSSION TITLE
Disable HTTP caching

### DISCUSSION THREAD

**Discussion by protocolist** [layday/instawow#125] [Q&A]
I'm trying to configure my addon GitHub release appropriately, and my release.json had some errors.

Running instawow install against the repo URL with -d -d option show that the HTTP responses from the GitHub API have been cached.

Is there a way I can disable this caching for testing purposes?

Many thanks

**Comment by layday** [comment]
Not from the command line I'm afraid but you can nuke the cache, run `instawow configure --show-active` and delete the `temp_dir`.  I'll see if I can add a flag for that in the coming days.

**Reply by protocolist** [reply]
Perfect, thank you
